opencti-platform/opencti-front/src/private/components/events/Incidents.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { FunctionComponent, useContext } from 'react';
+import { useViewStorage,
+} from '../../../utils/ListParameters';
+import ListCards from '../../../components/list_cards/ListCards';
+import ListLines from '../../../components/list_lines/ListLines';
+import IncidentsCards, {
+  incidentsCardsQuery,
+} from './incidents/IncidentsCards';
+import IncidentsLines, {
+  incidentsLinesQuery,
+} from './incidents/IncidentsLines';
+import IncidentCreation from './incidents/IncidentCreation';
+import Security from '../../../utils/Security';
+import { KNOWLEDGE_KNUPDATE } from '../../../utils/hooks/useGranted';
+import { UserContext } from '../../../utils/hooks/useAuth';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+import {
+  IncidentsLinesPaginationQuery,
+  IncidentsLinesPaginationQuery$variables,
+} from './incidents/__generated__/IncidentsLinesPaginationQuery.graphql';
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import { Filters } from '../../../components/list_lines';
+import { LOCAL_STORAGE_KEY_CASE } from '../cases/Incidents';
+import { IncidentLineDummy } from './incidents/IncidentLine';
+import { IncidentCardDummy } from './incidents/IncidentCard';
+import { IncidentsCardsPaginationQuery } from './incidents/__generated__/IncidentsCardsPaginationQuery.graphql';
+
+const Incidents: FunctionComponent = () => {
+  const { helper } = useContext(UserContext);
+  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<IncidentsLinesPaginationQuery$variables>(
+    LOCAL_STORAGE_KEY_CASE,
```

Comment:
events/incidents and cases/incidents are not the same object types ! you should redefine a key for Incidents.